bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/Rule.java

```diff
@@ -345,7 +353,7 @@ public Module getModule(String moduleId) {
         } else if (Action.class == moduleClazz) {
             result = (List<T>) actions;
         }
-        return result != null ? result : new ArrayList<T>(11);
+        return result != null ? result : Collections.<T> emptyList();
```

Comment:
Why not use consequently return Collections.emptyXXX()? not initialized collections? There is no need to set the triggers/actions/conditions to an empty (modifiable) in the getter. Same for the config.